resources/ansible_community_code_of_conduct.md

```diff
@@ -0,0 +1,86 @@
+## Ansible Community Code of Conduct
```

Comment:
It looks like this is the second copy of the Code of Conduct here in this PR. Again, I would link to the existing copy of the CoC in the documentation to avoid duplication and maintenance burdens.